diagnostics/setup_test.go

```diff
@@ -0,0 +1,25 @@
+package diagnostics_test
+
+import (
+	"testing"
+
+	"github.com/ledgerwatch/erigon/diagnostics"
+)
+
+func TestSetupEndpoints(t *testing.T) {
```

Comment:
not needed, ive disabled the sonar quality gate for code coverage, please revert

Reply:
Done